jsk_perception/sample/sample_mask_rcnn_instance_segmentation.launch

```diff
@@ -60,6 +60,8 @@
     <remap from="~input/label" to="apply_mask_image/output" />
     <rosparam subst_value="true">
       bg_label: -1
+      approximate_sync: True
+      slop: 1.0
```

Comment:
```suggestion
      approximate_sync: $(arg approximate_sync)
      slop: $(arg slop)
```

Reply:
and add these at the top
```
<arg name="approximate_sync" default="true" />
<arg name="slop" default="1.0" />
```

Reply:
Thank you very much. The code has been updated to reflect your suggestions.